internal/status/status.go

```diff
@@ -0,0 +1,20 @@
+package status
+
+import (
+	"runtime"
+	"soarca-gui/models/status"
+	"soarca-gui/utils"
+	"time"
+)
+
+var internalStatus = status.Status{Uptime: status.Uptime{Since: time.Now(), Milliseconds: 0},
+	Mode:    utils.GetEnv("LOG_MODE", "production"),
+	Runtime: runtime.GOOS}
+
+func SetVersion(version string) {
```

Comment:
Now everyone can set the version ...

Reply:
I copied this from SOARCA. so how to make this better?